### ISSUE TITLE
Enhance activity app backend to collect and display more information about event change

### ISSUE THREAD

**Issue by cm-schl** [nextcloud/server#23443]
Hello,

we've migrated our calendars from an old groupware to Nextcloud and are missing a function that the old system had: in Nextcloud Calendar and the Activity Stream it isn't possible to see some kind of changelog of single events. As shared calendars often are used to reserve rooms, cars or other resources by many users today it isn't possible to see the history of changes to a calendar entry.

My idea was to open a feature request in nextcloud/calendar (https://github.com/nextcloud/calendar/issues/2636) as some information (user, event name, calendar name and time of change) are already visible in the activity stream but are not shown when I open the detail view of a single event. Because the existing information does not seem to give back a complete changelog what happened to a event I was asked to open an issue here.

Ideally this information should be logged in the activity stream and thereby should thereby be accessible by the calendar app:

Creation of an event (already shown in activity stream):
- User
- Date and Time

Modification of an event (partially shown in activity stream at the moment):
- User
- Date and Time
- Difference between the old and new version of the event (for example: modification of start / end time, changes made to the name of the event, the description field, invitation ecc.).

**Comment by szaimen** [comment]
cc @nextcloud/server-triage is this feasible?

**Comment by cm-schl** [comment]
Hi,
this request is still valid as the option to see who created / edited a calendar entry would help to keep track of modifications in bigger environments.

**Comment by juliushaertl** [comment]
Makes sense I'd say. Any objections @ChristophWurst as that seems more like a groupware topic.

**Comment by cm-schl** [comment]
This feature request is still valid.
